PowerSimulationsPlots/src/report_design/report_design.jmd

```diff
@@ -1,18 +1,23 @@
 ---
 title : Weave example
 author : SIIP
-date : Friday the 13th
+date : 1-14
```

Comment:
Can we use `YYYY-MM-DD` for the format of dates everywhere?

Reply:
@kdheepak this is a report template for the user to change, so the date added was somewhat arbitrary

Reply:
Moved to another repo.